src/agent/tests/CMakeLists.txt

```diff
@@ -4,7 +4,9 @@ add_executable(agent_test agent_test.cpp)
 configure_target(agent_test)
 target_include_directories(agent_test PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/../include ${CMAKE_CURRENT_SOURCE_DIR}/../src)
 target_link_libraries(agent_test PRIVATE Agent GTest::gtest GTest::gmock)
+if(NOT WIN32)
```

Comment:
Why is this?

Reply:
At this stage the module for windows is not functional so we decided to discard the tests, which will be included in the issue [Support Compilation for Inventory Module on Windows Using MSVC](https://github.com/wazuh/wazuh-agent/issues/167#top)